src/libstd/sys/cloudabi/mutex.rs

```diff
@@ -57,15 +58,15 @@ impl ReentrantMutex {
     }
```

Comment:
Can you make this use `UnsafeCell::new(MaybeUninit::uninit())` (twice) instead?

Reply:
Should be fixed now!